serverless-map-reduce/amr.js

```diff
@@ -0,0 +1,81 @@
+const request = require('request-promise-native');
+const Redis = require('ioredis');
+
+/**
+ * Post an HTTP request to invoke a Binaris function.
+ *
+ * @param {string} [fname] Name of function to invoke
+ * @param {object} [fargs] Input arguments
+ *
+ * @return {object} Result returned from the function
+ */
+const invoke = async (fname, fargs) => {
+  console.log(`Invoking ${fname}()`);
+  const res = await request.post({
+    uri: `https://run.binaris.com/v2/run/${process.env.BINARIS_ACCOUNT_ID}/${fname}`,
+    json: true,
+    body: fargs
+  });
+  console.log(`${fname}() -> ${res}`);
+  return res;
+};
+
+// Connect to Redis
+const { REDIS_HOST, REDIS_PORT, REDIS_PASSWORD } = process.env;
+const redis = new Redis({
+  host: REDIS_HOST,
+  port: REDIS_PORT,
+  password: REDIS_PASSWORD,
+  tls: { rejectUnauthorized: false },
```

Comment:
Drop `rejectUnauthorized: false` please, it' bad practice and not supported by binaris.